CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.13)
+cmake_policy(VERSION 3.13)
+
+##############################################################################################################
+
+project(fty-common-dto VERSION 1.0.0)
+
+########################################################################################################################
+
+find_package(fty-cmake PATHS ${CMAKE_BINARY_DIR})    
+
+########################################################################################################################
+
+find_package(Protobuf REQUIRED)
+
+#Create the target
+etn_target(shared ${PROJECT_NAME}
+    SOURCES
+        src/*.cc
+    PUBLIC_INCLUDE_DIR
+        include
+    PUBLIC
+        *.h
+    USES_PUBLIC
+        cxxtools
```

Comment:
do not use not cmake based projects as public dependencies :)

Reply:
Even for pkgconfig project ?